community.html

```diff
@@ -71,6 +71,19 @@ <h2><i class="fa fa-twitter"></i> Scrapy tweets</h2>
     </p>
   </div>
 
+  <div class="community-box">
+    <a href="https://t.me/scrapy_python">
+      <h2><i class="fa fa-paper-plane"></i>Telegram community</h2>
+    </a>
+    <hr />
+    <p>
+      You can join the <span class="highlight">Telegram community</span>
+      to chat with other Scrapy users and developers,
+      via <a href="https://t.me/scrapy_python">link</a>.
+      There are mostly Russian-speaking devs.
```

Comment:
I see that currently all active members are messaging each other in Russian. Do we want this to change? Or should we instead feature this as the Russian-speaking Telegram group, and only encourage other Russian-speaking people to join?

Reply:
I'd feature it as a Russian-speaking Telegram group, hence my previous suggestion to rename it to "Russian Telegram community".

We should start a Spanish one btw ;)